.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,105 @@
+name: Bug Report Form
+description: Report an issue related to the Home Assistant Operating System.
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Make sure to test with the last version of the Operating System
+        before reporting a bug. If the bug appears to be a regression,
+        make sure to check if the bug indeed disappears from the
+        previous version. Use `ha os update --version x.y` command to
+        downgrade.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the issue you are experiencing
+      description: Provide a clear and concise description of what the bug is.
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: What operating system image do you use?
+      options:
+        - generic-x86-64 (Generic UEFI capable x86-64 systems)
+        - khadas-vim3 (Khadas VIM3)
+        - odroid-c2 (Hardkernel ODROID-C2)
+        - odroid-c4 (Hardkernel ODROID-C4)
+        - odroid-n2 (Hardkernel ODROID-N2/N2+)
+        - odroid-xu4 (Hardkernel ODROID-XU4)
+        - ova (for Virtual Machines)
+        - rpi (Raspberry Pi 1)
+        - rpi0-w (Raspberry Pi Zero W)
+        - rpi2 (Raspberry Pi 2)
```

Comment:
These builds are not supported, should they even be here?
https://github.com/home-assistant/architecture/blob/master/adr/0015-home-assistant-os.md#supported-boardshardwaremachines

Reply:
:man_shrugging: I mean I don't test them and all, but if there is something small wrong with them, I would just fix it. My thinking was that I can still reject it on the ground of its not officially supported.

I'd say if we don't allow creating issues for them, we almost have to stop building them even.

Reply:
Honestly.. I think we should stop

Reply:
Agreed.

Reply:
```json
{
  "rpi3-64": 7055,
  "ova": 17534,
  "rpi4-64": 18830,
  "rpi3": 2576,
  "rpi4": 1935,
  "odroid-n2": 2317,
  "generic-x86-64": 2549,
  "rpi0-w": 106,
  "tinker": 60,
  "rpi2": 286,
  "odroid-c2": 42,
  "odroid-c4": 21,
  "odroid-xu4": 46,
  "intel-nuc": 25,
  "rpi": 25
}
```

I mean, at least the rpi2 seems quite popular still.

I think I'd phase out the ARMv6 boards first, so `rpi0-w` and `rpi`. What we also could do is to stop offer images for new installations and continue to offer `raucb` for a version or so.

I am amazed how far along the `intel-nuc` -> `generic-x86-64`

Reply:
I agree with Stefan, we should first face out the armv6 support. RPi2 is popular, there is no technical reason for not build an image

Reply:
I would also not stop distributing rpi2 images for now as this is still quite a widely used platform. But with the pi zero 2 w out (which can directly use a rpi3 image) consolidation will come up in the somewhat near future, yes. 

Before that, however, there is IMHO no reason to build 32bit images for rpi3+rpi4 anymore or did I miss something here? Why are we still building 32bit versions for rpi3/rpi4?

Reply:
Ok, let's drop `rpi` `rpi0-w` for now: https://github.com/home-assistant/operating-system/pull/1638

> But with the pi zero 2 w out (which can directly use a rpi3 image

Interesting, I was thinking about adding the Pi Zero 2, but don't have one yet. So you tested it, works with the rpi3 board?

> Before that, however, there is IMHO no reason to build 32bit images for rpi3+rpi4 anymore or did I miss something here? Why are we still building 32bit versions for rpi3/rpi4?

Yeah agreed, we could phase those out slowly now. I guess its still a bit early at this point We have 2k+ active installation on each one of them according to analytics. It would lead to quite some churn for people as well. I think we need to start with removing it from installation instructions etc.